tests/functional/pv/pvc_snapshot/test_rbd_cbt_metadata.py

```
@@ -414,3 +422,223 @@ def test_cbt_delta_incremental(self, volume_mode):
+        ), "caCert does not end with PEM footer"
+
+        # Parse the certificate to verify it's valid
+        try:
+            cert = x509.load_pem_x509_certificate(
+                ca_cert.encode("utf-8"), default_backend()
+            )
+            log.info(
+                "caCert is a valid PEM-encoded certificate (Subject: %s)",
+                cert.subject.rfc4514_string(),
+            )
+        except Exception as e:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)